### ISSUE TITLE
start page

### ISSUE THREAD

**Issue by lcabrini** [lcabrini/aluguntugui#4]
The site will need to have a start page from which you should be able to sign up and sign in. I don't know what else should go on to this page yet.

**Comment by lcabrini** [comment]
It seems most language learning sites start off by asking you which language you are interested in learning. Should we do something similar, or do should we try something different?

**Comment by benbaka** [comment]
When they ask, does it end up as part of User registration information or what ? If you can find out the reason they require it then that could help us incorporate it into ours with a better understanding.

**Comment by lcabrini** [comment]
Yes, in most cases it does. However, on some sites, like duolingo, you could use the site without being registered. Just that you won't get your XP. 

However, I think that since the first milestone release and we don't have a language list yet, I think we should just create a start page with some placeholder text for the time being. More important is to get the registration and login buttons on the page so that we can actually test that functionality. As well as links to other pages such as About, Contact, Privacy, etc. of course.
